mergify_engine/engine/actions_runner.py

```diff
@@ -164,6 +197,8 @@ async def gen_summary(
             summary += STRICT_MODE_DEPRECATION_SASS
         else:
             summary += STRICT_MODE_DEPRECATION_GHES
+    if _has_merge_action_with_commit_message_mode(pull_request_rules) is not None:
+        summary += _has_merge_action_with_commit_message_mode(pull_request_rules)
```

Comment:
You're calling `_has_merge_action_with_commit_message_mode`.

Imagine the function takes 2 hours… now it takes 4 hours to run this code 😢